rllib/examples/connectors/classes/euclidian_distance_based_curiosity.py

```diff
@@ -43,6 +43,8 @@ class EuclidianDistanceBasedCuriosity(ConnectorV2):
     If you would like to use a simpler, count-based mechanism for intrinsic reward
     computations, take a look at the `CountBasedCuriosity` connector piece
     at `ray.rllib.examples.connectors.classes.count_based_curiosity`
+
+    This connector piece doesn't alter the incoming `batch` and returns it as-is.
```

Comment:
Sweet! I didn't know we had this implemented. This corresponding logic exist under the old stack at some point?